cgi-bin/LJ/Widget/UserpicSelector.pm

```diff
@@ -21,8 +21,10 @@ use LJ::Talk;
 
 sub need_res {
 
-    # force to not use beta, because this is not used in journal spaces
-    return LJ::Talk::init_iconbrowser_js( 0, 'stc/entry.css' );
+    # Just let need_res work normally for this, instead of using LJ::Widget's
+    # berserk reimplementation of it.
+    LJ::Talk::init_iconbrowser_js('stc/entry.css');
```

Comment:
Actually, looking further at this, I think the whole "take extra includes as arguments" thing is ALSO just an artifact of the weird widget thing. Can probably remove that too, have init_iconbrowser take no args, and just return that extra stc/entry.css include from this method for the normal widget need_res system to deal with.